readthedocs_ext/readthedocs.py

```diff
@@ -338,6 +338,25 @@ def dump_sphinx_data(app, exception):
         )
 
 
+def dump_telemetry(app, config):
+    # We need to get the output directory where the docs are built
+    # _build/json.
+    build_json = os.path.abspath(
+        os.path.join(app.outdir, '..', 'json')
+    )
+    outjson = os.path.join(build_json, 'telemetry.json')
+    outdir = os.path.dirname(outjson)
+    if not os.path.exists(outdir):
+        os.makedirs(outdir)
+    with open(outjson, 'w+') as json_file:
```

Comment:
Should we have some kind of exception handling here, since this is running inside of the doc build, we basically never want an exception to stop the build for some reason.